### ISSUE TITLE
Plans for making this package available for R 4.0.2

### ISSUE THREAD

**Issue by PublicHealthDataGeek** [Robinlovelace/ukboundaries#7]
Hello

Thanks for this fab package.  Do you know if you are planning of releasing it for R 4.0.2?

**Comment by Robinlovelace** [comment]
Does it not install with R 4.0.2?

**Comment by PublicHealthDataGeek** [comment]
Not according to my R.. :(

``` r
install.packages("ukboundaries")
#> Installing package into '/home/bananafan/R/x86_64-pc-linux-gnu-library/4.0'
#> (as 'lib' is unspecified)
#> Warning: package 'ukboundaries' is not available (for R version 4.0.2)
```

<sup>Created on 2020-08-06 by the [reprex package](https://reprex.tidyverse.org) (v0.3.0)</sup>

**Comment by PublicHealthDataGeek** [comment]
Apologies, I would describe that as 'user error'! Closing!!!!!
